CucumberTest/src/test/java/com/knight/cucumber/Runner.java

```diff
@@ -0,0 +1,19 @@
+package com.knight.cucumber;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import cucumber.api.junit.Cucumber;
+import cucumber.api.CucumberOptions;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(features = "src/test/resources/features/", glue = "com.knight.cucumber",  
+plugin = {"pretty","html:E:/eclipse-new-ws/CucumberTest/test-output","json:E:/eclipse-new-ws/CucumberTest/json-output/c.json","junit:E:/eclipse-new-ws/CucumberTest/xml-output/c.xml"},

```

Comment:
1. code is not properly formatted
2. use of local machine path should be avoided. it should be inside project or any server path